### PR TITLE
Added routine to update workspace size when settings are changed.

### PR DESCRIPTION
**Pull request by johnbenweber**
When either bed width or bed height is changed the workspace bounds drawn on the canvas get resized to reflect the changes.

### CODE REVIEW

File: UIElements/gcodeCanvas.py [hunk 1]
```diff
@@ -84,6 +67,25 @@ def zoom(self, callback, type, motion, *args):
...
+
+    def drawWorkspace(self, *args):
+
+        self.scatterObject.canvas.remove_group('workspace')
+ 
+        with self.scatterObject.canvas:
+            Color(1, 1, 1)
+
+            #create the bounding box
+            height = float(self.data.config.get('Maslow Settings', 'bedHeight'))
```
Comment (BarbourSmith) [inline UIElements/gcodeCanvas.py]: 👍

**Comment by BarbourSmith** [comment]
Two pull requests merged before breakfast! 🔥 🔥 🔥 

This one has some conflicts with the first one, but other than that it looks fantastic. 

I think it's better if you resolve them because you have a better understanding of what changed.

Great work!

**Comment by johnbenweber** [comment]
Yeah, should have waited until that gcode cleanup pull request got approved. Anyway I tested it and I think it's all good now, but you probably want to give it a good test before approving.

**Comment by BarbourSmith** [comment]
Looks great!

**Pull request merged**